get-started/code-experiments/README.md

```diff
@@ -0,0 +1,260 @@
+## DVC Get Started for Experiments
+
+This repository contains a project that trains a classifier for [Fashion
+MNIST][fmnist] dataset in Tensorflow. 
+
+[fmnist]: https://github.com/zalandoresearch/fashion-mnist
+
+It is used as a showcase for [`dvc exp`][gsdvcexp] commands to manage large
+number of experiments. 
+
+[gsdvcexp]: https://dvc.org/doc/start/experiments
+
+After [installing DVC][installdvc] and cloning the repository, you can run:
+
+[installdvc]: https://dvc.org/doc/install
+
+```console
+python -m venv .venv
+. .venv/bin/activate
+python -m pip install -r requirements.txt
+```
+
+`params.yaml` defines many parameters to modify with `dvc exp run --set-param/-S`
+option. For example to set the number of convolutional units in `cnn` model, you
+can use:
+
+```console
+dvc exp run -S model.cnn.conv_units=128 
+```
+
+The experiment will produce a new `metrics.json` and you can check the changes
+in metrics:
+
+```console
+dvc exp diff
+```
+
+You can queue many experiments with `--queue` option:
+
+```console
+dvc exp run --queue -S model.cnn.dropout=0.1
+dvc exp run --queue -S model.cnn.dropout=0.2
+dvc exp run --queue -S model.cnn.dropout=0.3
+dvc exp run --queue -S model.cnn.dropout=0.4
+dvc exp run --queue -S model.cnn.dropout=0.5
+dvc exp run --queue -S model.cnn.dropout=0.6
+dvc exp run --queue -S model.cnn.dropout=0.7
+dvc exp run --queue -S model.cnn.dropout=0.8
+dvc exp run --queue -S model.cnn.dropout=0.9
+```
+
+and run them at once, preferably in parallel with `--jobs`:
+
+```console
+dvc exp run --run-all --jobs 4
+```
+
+You can get the summary of experiments with: 
+
+```console
+dvc exp show
+```
+
+and limit the parameters and metrics to show with `--include-params` and
+`--include-metrics` options, respectively.  
+
+Experiments are given hash-names derived from their inputs and environment. It
```

Comment:
"Hash-names?" Is this this official DVC term for this? CC @jorgeorpinel.

Also "derived from their inputs and environment" - can we be more specific or link to documentation explaining the naming convention?

Reply:
https://dvc.org/doc/command-reference/exp/run#options are you looking for `dvc exp run --name <name>`?

Reply:
The autogenerated names are based on a SHA256 hash of the reproduced pipeline. They are hashed in the same way that we generate hashes for stages which are used in run-cache, except that we hash the entire pipeline instead of individual stages.

The hashes really only have meaning internally to DVC, I'm not sure it needs to be documented anywhere outside of the DVC code base itself. If users want experiment names to mean something they should be providing their own names via `-n/--name`.

Reply:
> Is this this official DVC term for this?

We just use "names" now. See https://dvc.org/doc/command-reference/exp/run. I see no need to go into any details on how they're named here TBH.

```suggestion
Experiments are given automatic names but it
```